src/firefly/js/charts/ui/options/SpectrumOptions.jsx

```diff
@@ -310,7 +310,7 @@ export function spectrumInputs ({chartId, groupKey}) {
                 : <ValidationField fieldKey={SFOptionFieldKeys(activeTrace).value} initialState={{value: sfRefPos}} readonly={true} {...allProps}/>;
         },
     };
-}
+}, (props) => Object.values(props).join('|'));
```

Comment:
might be worth defining this repeated resolver function in BasicOptions and reusing it from there?